client/src/components/AuditAdmin/Setup/Review/StandardizeContestChoiceNamesDialog.tsx

```diff
@@ -0,0 +1,244 @@
+import {
+  Button,
+  Callout,
+  Classes,
+  Colors,
+  Dialog,
+  HTMLSelect,
+  HTMLTable,
+} from '@blueprintjs/core'
+import React, { useState } from 'react'
+import styled from 'styled-components'
+
+import FormButton from '../../../Atoms/Form/FormButton'
+import { IContest } from '../../../../types'
+import { IContestChoiceNameStandardizations } from '../../../useContestChoiceNameStandardizations'
+import { IJurisdiction } from '../../../useJurisdictions'
+
+export function isContestChoiceNameStandardizationComplete(
+  standardizations: IContestChoiceNameStandardizations
+): boolean {
+  for (const jurisdictionStandardizations of Object.values(standardizations)) {
+    for (const contestStandardizations of Object.values(
+      jurisdictionStandardizations
+    )) {
+      for (const standardizedChoiceName of Object.values(
+        contestStandardizations
+      )) {
+        if (standardizedChoiceName === null) {
+          return false
+        }
+      }
+    }
+  }
+  return true
+}
+
+const Table = styled(HTMLTable)`
+  background: ${Colors.WHITE};
+  border: 1px solid ${Colors.LIGHT_GRAY1};
+  width: 100%;
+
+  tr th,
+  tr td {
+    vertical-align: middle;
+    word-wrap: break-word;
+  }
+`
+
+interface IDialogProps {
+  contest: IContest
+  isOpen: boolean
+  jurisdictionsById: { [id: string]: IJurisdiction }
+  onClose: () => void
+  standardizations: IContestChoiceNameStandardizations
+  standardizedContestChoiceNames: string[]
+  updateStandardizations: (
+    newStandardizations: IContestChoiceNameStandardizations
+  ) => Promise<void>
+}
+
+/**
+ * A dialog for standardizing contest choice names
+ */
+export const StandardizeContestChoiceNamesDialog: React.FC<IDialogProps> = ({
+  isOpen,
+  onClose,
+  standardizations,
+  contest,
+  jurisdictionsById,
+  standardizedContestChoiceNames,
+  updateStandardizations,
+}) => {
+  const [formState, setFormState] = useState(standardizations)
```

Comment:
I considered using react-hook-form but chose not to for a few reasons:

1. Our field identifiers can have dots in them (because choice names can have dots, e.g., "Joe Jr.") which can throw off how react-hook-form accesses state. There are workarounds, but they're a bit convoluted.
2. This form is ultimately quite simple, and I found it easier to reason about manually than with a library